### PR TITLE
adding rust code example using CDRS driver

### PR DESCRIPTION
**Pull request by guy9**
@tzach please review

### CODE REVIEW

File: rust/ps-logger/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Scylla tutorial
```
Comment (tzach) [inline rust/ps-logger/README.md]: Please add more context to the README
what is this example doing?
See example in Python, Java lessons

File: rust/ps-logger/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+To get it up and running start a local scylla instance started on port 9042:
+```bash
+docker run   -p 9042:9042/tcp   --name some-scylla   --hostname some-scylla   -d scylladb/scylla:3.3.0    --smp 1 --memory=750M --overprovisioned 1
+```
+
+Then run the code:
+```bash
+$ cargo run
```
Comment (tzach) [inline rust/ps-logger/README.md]: This failed for me with 
```
error[E0432]: unresolved import `self::std::hint`
  --> /home/tzach/.cargo/registry/src/github.com-1ecc6299db9ec823/lazy_static-1.4.0/src/inline_lazy.rs:13:16
   |
13 | use self::std::hint::unreachable_unchecked;
   |                ^^^^ Could not find `hint` in `std`

   Compiling smallvec v0.6.10
error: aborting due to previous error

error: Could not compile `lazy_static`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by guy9** [comment]
Good catch @tzach, thanks. Please try again according to the updated README file.

**Comment by tzach** [comment]
> Good catch @tzach, thanks. Please try again according to the updated README file.

I already have cargo installed so the problem is different. I still see the same issue as a bove

**Comment by tzach** [comment]
Look like I used an older version of rust
running `rustup update` fixed the issues

**Comment by tzach** [comment]
Got results, but I have no idea what they means.
Will be good to add some wording on the example in the README

```
connecting to db
Adding measurements
Selecting measurements
     >> Measurements: [TemperatureMeasurement { device: 72f6d49c-76ea-44b6-b1bb-9186704785db, time: Timespec { sec: 1000000000001, nsec: 0 }, temperature: 40 }, TemperatureMeasurement { device: 72f6d49c-76ea-44b6-b1bb-9186704785db, time: Timespec { sec: 1000000000003, nsec: 0 }, temperature: 60 }]

```

**Comment by guy9** [comment]
Done

**Pull request merged**